bpf/lib/l3.h

```diff
@@ -81,8 +81,7 @@ l3_local_delivery(struct __ctx_buff *ctx, __u32 seclabel,
 		return redirect_ep(ctx, ep->ifindex, from_host);
 #endif /* DISABLE_LOOPBACK_LB */
 
-#if defined(USE_BPF_PROG_FOR_INGRESS_POLICY) && \
-	!defined(FORCE_LOCAL_POLICY_EVAL_AT_SOURCE)
+#if defined(USE_BPF_PROG_FOR_INGRESS_POLICY)
```

Comment:
This change will break the https://github.com/cilium/cilium/pull/22006 PR (as `bpf_redirect_peer` will bypass the `ingress` at the target `lxc`).

This is ok, however, to remove the option from the userspace, as we're setting the `FORCE_LOCAL_POLICY_EVAL_AT_SOURCE` macro inside `lib/l3.h`.

So this would be cool if you revert the `bpf/` part (in the context of this PR this is a non-functional code, as nobody will ever define the `FORCE_LOCAL_POLICY_EVAL_AT_SOURCE`)

Reply:
Thanks for context. I've dropped the changes in `bpf/` and amended the commit message accordingly.